linux/step05_centos7_nginx.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+#start-copy
+cp setup_omero_nginx.sh ~omero
+#end-copy
+
 #start-install
 yum -y --enablerepo=cr install nginx
```

Comment:
cr is not required. so it should be removed. The version installed will be 1.6.3. 
To match the recommended version cf. https://github.com/openmicroscopy/ome-documentation/pull/1439, we will have to follow the same approach that the one used for centos 6. 
For the rest of the dependencies, we have installed the recommended version
I will update the code according